application-services/custom/camera-management/README.md

```diff
@@ -1,340 +1,13 @@
-
-
-### Running Pipelines
-
-Once the pipeline is running, you can view the pipeline and its status.
-
-![default pipelines state](./images/multiple-pipelines-default.png)  
-
-1. Expand a pipeline to see its status. This includes important information aush as elapsed time, latency, frames per second, and elapsed time.  
-   ![select-camera](./images/running-pipelines.png)  
-
-1. In the terminal where you started the app, once the pipeline is started, this log message will pop up.
-    ```bash
-    level=INFO ts=2022-07-11T22:26:11.581149638Z app=app-camera-management source=evam.go:115 msg="View inference results at 'rtsp://<SYSTEM_IP_ADDRESS>:8555/<device name>'"
-    ```
-
-1. Use the URI from the log to view the camera footage with analytics overlayed.
-    ```bash
-    ffplay 'rtsp://<SYSTEM_IP_ADDRESS>:8555/<device name>'
-    ```
-
-   Example Output:  
-   ![example analytics](./images/example-analytics.png)  
-
-1. Press the red square stop button to shut down the pipeline.
-
-
-### API Log
-
-The API log shows the status of the 5 most recent calls and commands that the management has made. This includes important information from the responses, including camera information or error messages.
-
-![API Logs](./images/api-log.png)  
-
-1. Expand a log item to see the response
-
-   Good response:
-   ![good api response](./images/good-response.png)  
-   Bad response:
-   ![bad api response](./images/bad-response.png)  
-
-### Inference Events
-
-![inference events default](./images/inference-events-default.png)  
-
-1. To view the inference events in a json format, click the `Stream Events` button.
-
-![inference events](./images/inference-events.png)  
-
-### Inference results in Edgex
-
-To view inference results in Edgex, open Edgex UI [http://localhost:4000](http://localhost:4000), click on the `DataCenter`
-tab and view data streaming under `Event Data Stream`by clicking on the `Start` button.
-
-![inference events](./images/inference-edgex.png)
-
-### Next steps
-A custom app service can be used to analyze this inference data and take action based on the analysis.
-
-## Additional Development
-
-> **Warning**: The following steps are only useful for developers who wish to make modifications to the code
-> and the Web-UI.
-
-#### Development and Testing of UI
-##### 1. Build the production web-ui
-This builds the web ui into the `web-ui/dist` folder, which is what is served by the app service on port 59750.
-```shell
-make web-ui
-```
-
-##### 2. Serve the Web-UI in hot-reload mode
-This will serve the web ui in hot reload mode on port 4200 which will recompile and update anytime you make changes to a file. It is useful for
-rapidly testing changes to the UI.
-```shell
-make serve-ui
-```
-
-Open your browser to [http://localhost:4200](http://localhost:4200)
+> The **main** branch of this repository contains work-in-progress development code for the upcoming release, and is **not guaranteed to be stable or working**.
```

Comment:
I don't think this warning is applicable to the example app (atleast for now as in future we do want create releases as other repos and use main branch as development branch). In the edgex docs we are asking the user to create the example app docker image from the main branch only.

Reply:
So do we want users to use main for the examples repo and minnesota for everything else?

Reply:
Examples repo is [tagged](https://github.com/edgexfoundry/edgex-examples/releases/tag/v3.0) like all other repos and moving forward main will be WIP like other repos.

Reply:
In that case @ChristianDarr-personal you might want to add this point in edgex docs to checkout the latest stable tag and then create the docker image for the example app.

Reply:
Ok, so it sounds like the original warning still applies, and then we point them to see the most recent tagged release